Sources/ProjectSpec/PlatformVersions.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  Version.swift
+//  ProjectSpec
+//
+//  Created by Yonas Kolb on 22/12/17.
+//
+
+import Foundation
+import xcproj
+import JSONUtilities
+
+public struct PlatformVersions: Equatable {
+
+    public var iOS: String?
+    public var tvOS: String?
+    public var watchOS: String?
+    public var macOS: String?
+
+    public init(iOS: String? = nil, tvOS: String? = nil, watchOS: String? = nil, macOS: String? = nil) {
+        self.iOS = iOS
+        self.tvOS = tvOS
+        self.watchOS = watchOS
+        self.macOS = macOS
+    }
+
+    public func version(for platform: Platform) -> String? {
+        switch platform {
+        case .iOS: return iOS
+        case .tvOS: return tvOS
+        case .watchOS: return watchOS
+        case .macOS: return macOS
+        }
+    }
+
+    public static func == (lhs: PlatformVersions, rhs: PlatformVersions) -> Bool {
+        return lhs.iOS == rhs.iOS &&
+            lhs.tvOS == rhs.tvOS &&
+            lhs.watchOS == rhs.watchOS &&
+            lhs.macOS == rhs.macOS
```

Comment:
This will probably be fine but will fail for instances where you might compare "11" and "11.0" since we are dealing with string equality vs a structured representation of a semantic version

Reply:
That's true. We could potentially implement a proper Version type (or use a dependency, like Swift PM), but at the moment it's probably fine

Reply:
I think you could copy this implementation that I made for Sake https://github.com/xcodeswift/sake/blob/master/Sources/SakefileUtils/Version.swift that has the equatable defined. That'd solve this issue easily.

Reply:
Done. One change I had to make was have a getter on that version type that returns an Apple deployment version, as they use `9.0` not `9.0.0`